### DISCUSSION TITLE
Add getInfo to context.app on the Plugin API

### DISCUSSION THREAD

**Discussion by monrax** [Kong/insomnia#3994] [Ideas]
**Is your feature request related to a problem? Please describe.**
There is no way to get information about the Insomnia application, like the version.

**Describe the solution you'd like**
Create a getInfo method for context.app that returns an object with app info.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by dimitropoulos** [comment]
could you say a little more about what your use-case is?  Is there anything other than `version` that you'd like to have the info object contain?

**Reply by monrax** [reply]
Hi, sorry for the late reply.  I only need the app `version`, I just thought that it was better if the `getInfo` method returned more info than that. @develohpanda has explained to me why exposing things that are not entirely useful might not be the best course of action. I was going to remove the other three today, but it seems Opender beat me to it!

**Reply by develohpanda** [reply]
This is the unfortunate thing with GH discussions at the moment 😔 @dimitropoulos there is an open PR: #3975
